npg_qc_viewer/t/client/test_qc_outcomes_view.js

```diff
@@ -283,6 +298,64 @@ requirejs(['scripts/qcoutcomes/qc_outcomes_view', 'scripts/qcoutcomes/qc_css_sty
       }
     });
 
+    QUnit.test("Tests correct number & type of parameters in usabilityDisplaySwitch", function (assert) {
+      assert.throws(
+        function() {
+          mqc_outcomes.usabilityDisplaySwitch();
+        },
+        /parameter obj must be defined/i,
+        'Throws with no params'
+      );
+      assert.throws(
+        function() {
+          mqc_outcomes.usabilityDisplaySwitch(1);
+        },
+        /usability should be boolean/i,
+        'Throws with one param'
+      );
+      assert.throws(
+        function() {
+          mqc_outcomes.usabilityDisplaySwitch(1, 'string');
+        },
+        /usability should be boolean/i,
+        'Throws with one string instead of boolean'
+      );
+    });
+
+    QUnit.test("Test display of usability REJECTED icon in LaneNo column ", function (assert) {
+      var laneCellsNumb = $('td.lane').length;
+      assert.equal(laneCellsNumb, 9, 'Correct number of .lane cells');
+      var total_images = $("td.lane img").length;
```

Comment:
Not testing for images any more. Should be something like $("td.lane > div.utility_fail") but make sure

Reply:
Variable name as total_utility_fails?